binding.gyp

```diff
@@ -117,12 +159,16 @@
       "./src/image.cc",
       "./src/keyboard.cc",
       "./src/mouse.cc",
+      "./src/joystick.cc",
```

Comment:
nit: I think we should put them as alphabetical order except `./src/sfml.cc`.

Reply:
Good for me, I fixed the order :)